processor/src/main/java/ca/bc/gov/app/service/legacy/LegacyIndividualPersistenceService.java

```diff
@@ -109,19 +109,12 @@ public Mono<MessagingWrapper<ForestClientDto>> generateForestClient(
                             )
                         )
                     )
-                    .withClientIdentification(
-                        Objects.toString(
-                            detailEntity.getIdentificationCode(),
-                            ProcessorUtil.splitName(
-                                getUser(message, ApplicationConstant.CREATED_BY)
-                            )[0]
-                        )
-                    )
+                    .withClientIdentification(detailEntity.getClientIdentification())
```

Comment:
This change only takes into account the staff-submitted form. The older code will check for the identification that will come in case of an individual on staff, and in case of external bcsc, this field will be empty, thus falling back to the user id.

For this to work, the code for the BCSC should set the ID on that field and I'm not seeing it here